elytron/src/main/java/org/wildfly/extension/elytron/PrincipalDecoderDefinitions.java

```diff
@@ -247,30 +258,48 @@ public void registerAttributes(ManagementResourceRegistration resourceRegistrati
 
     private static class PrincipalDecoderAddHandler extends BaseAddHandler {
 
+        private static final Set<RuntimeCapability> CAPABILITIES;
+
+        static {
+            Set<RuntimeCapability> capabilities = new HashSet<>(2);
+            capabilities.add(PRINCIPAL_DECODER_RUNTIME_CAPABILITY);
+            capabilities.add(PRINCIPAL_TRANSFORMER_RUNTIME_CAPABILITY);
+            CAPABILITIES = capabilities;
+        }
 
         private PrincipalDecoderAddHandler(AttributeDefinition ... attributes) {
-            super(PRINCIPAL_DECODER_RUNTIME_CAPABILITY, attributes);
+            super(CAPABILITIES, attributes);
```

Comment:
PrincipalDecoderDefinitions is being removed entirely after the next tag, this is just an incremental change to minimise impact to the console.  Debugging another issue we realise the principal decoders are an obsolete concept.